app/src/templates/dashboard.html

```diff
@@ -119,39 +119,16 @@ <h4>Test Yourself!</h4>
                 <div class="card-body3">
                     <div id="leaderboard-card-header">
                         <h4>Weekly Leaderboard</h4>
-                        <!-- <a href="{{ leaderboard_url }}" id="leaderboard-link">
-                            
-                        </a> -->
-                        <!-- put the button line inside the <a> tag above after we set up the dashboard route to send in 
-                            the leaderboard info -->
-                        <button id="view-leaderboard">VIEW ALL</button>
+                        <button id="view-leaderboard" onclick="window.location.href='/leaderboard'">VIEW ALL</button>
                     </div>
                     <ul id="leaderboard-list">
+                        {% for user in leaderboard_data %}
```

Comment:
Looks good, just wanted to note that the dashboard route should limit the query for leaderboard_data to the top 5 users if we haven't set that up yet.